.gitignore

```diff
@@ -1,5 +1,7 @@
 __pycache__
 
+cpp/.vs
```

Comment:
Don't need this as it's system specific config.

Reply:
I don't mind having this here. Yes, it is system-specific, but at the same time, I'd like for Visual Studio to be an "officially supported" way of debugging on Windows.

At the same time, I don't want to maintain IDE projects for every platform, but I'm ok with ignoring vs temp files.